index.js

```diff
@@ -69,30 +71,18 @@ module.exports = opts => {
     'verbose': false
   };
 
-  const options = { ...defaultOptions, ...opts };
+  const mergedOptions = { ...defaultOptions, ...options };
 
   // Set options
-  for (const key of Object.keys(options)) {
-    const value = options[key];
+  for (const key of Object.keys(mergedOptions)) {
+    const value = mergedOptions[key];
     if (key === 'format' && value !== 'gnu') {
-      vnuArgs = [...vnuArgs, `--format ${value}`];
+      vnuArgs.push('--format', value);
```

Comment:
Wasn't this wrong before? It triggers a new unsafe shell command error. Any ideas how to fix this welcome.

Reply:
I pushed a patch which limits the format values, let me know what you think